iree/tools/init_iree_dialects.h

```diff
@@ -37,6 +38,7 @@ inline void registerIreeDialects(DialectRegistry &registry) {
                   IREE::Flow::FlowDialect,
                   IREE::HAL::HALDialect,
                   IREE::LinalgExt::IREELinalgExtDialect,
+                  mlir::linalg::transform::LinalgTransformDialect,
```

Comment:
Ah, I know why this is here, but the namespace difference is a bit off-putting. Just noting, not making it blocking.